apps/judicial-system/web/src/utils/utils.spec.ts

```diff
@@ -448,4 +450,85 @@ describe('Step helper', () => {
       expect(ind).toEqual(false)
     })
   })
+
+  describe('removeTabs', () => {
+    test('should replace a single tab with a single space', () => {
+      // Arrange
+      const str = '\t'
+
+      // Act
+      const res = replaceTabs(str)
+
+      // Assert
+      expect(res).toEqual(' ')
+    })
+
+    test('should replace multiple consecutive tabs with a single space', () => {
+      // Arrange
+      const str = '\t\t\t'
+
+      // Act
+      const res = replaceTabs(str)
+
+      // Assert
+      expect(res).toEqual(' ')
+    })
+
+    test('should remove multiple consecutive tabs with a leading space', () => {
+      // Arrange
+      const str = ' \t\t\t'
+
+      // Act
+      const res = replaceTabs(str)
+
+      // Assert
+      expect(res).toEqual(' ')
+    })
+
+    test('should remove multiple consecutive tabs with a trailing space', () => {
+      // Arrange
+      const str = '\t\t\t '
+
+      // Act
+      const res = replaceTabs(str)
+
+      // Assert
+      expect(res).toEqual(' ')
+    })
+
+    test('should process a complicated string with tabs', () => {
+      // Arrange
+      const str =
+        'Lorem\t ipsum dolor \t\tsit amet,\t\t\t\tconsectetur \t\t\t adipiscing elit.'
```

Comment:
Viljum við að  \t\t\t verði tvö "  " en ekki eitt " "?

Reply:
Hugsunin hjá mér er að breyta bara tabs, ekki spaces. Hins vegar hafa space við hliðina á tabs áhrif á hvað kemur í staðinn fyrir tabs.

Reglan er þessi:
Röð (1+) af tabs sem er ekki við hliðina á space, verður að einu space.
Röð (1+) af tabs sem er við hliðina á space, verður að engu.

Thats it, engar aðrar reglur.

Í dæminu verður því ' \t\t\t ' að '  ', þ.e. space+tab+tab+tab+space verður að space+space.